boa/src/builtins/date/mod.rs

```diff
@@ -217,15 +217,15 @@ impl Date {
     /// Converts the `Date` to a local `DateTime`.
     ///
     /// If the `Date` is invalid (i.e. NAN), this function will return `None`.
-    pub fn to_local(&self) -> Option<DateTime<Local>> {
+    pub fn as_local(&self) -> Option<DateTime<Local>> {
```

Comment:
As a convention, we are keeping the original JS name in Rust too. In this case, `to_local()`. This gives some `clippy` errors/warnings, but we are just adding `#[allow(clippy::lint_name)]` for these cases. Either we do the same here, or we change it everywhere else, but we should be consistent.

Reply:
I’m fully open to that, I can set ignores here

Reply:
That would be my preferred option too :)